src/recovery/kafka.rs

```diff
@@ -169,6 +172,25 @@ pub(crate) fn create_kafka_topic(brokers: &[String], topic: &str, partitions: i3
     }
 }
 
+fn to_bytes<T>(obj: &T) -> Vec<u8>
```

Comment:
These are now only needed for Kafka recovery, since SQLite recovery has a reasonable type for each column.